test/distributed/org/apache/cassandra/fuzz/topology/HarryTopologyMixupTest.java

```diff
@@ -164,90 +216,49 @@ private static CommandGen<Spec> cqlOperations(Spec spec)
 
     private static Command<State<Spec>, Void, ?> validateAll(State<Spec> state)
     {
-        Spec spec = state.schemaSpec;
-        var schema = spec.harry.schema();
-        boolean writeThroughAccord = schema.isWriteTimeFromAccord();
+        Spec spec = state.testState;
         List<Command<State<Spec>, Void, ?>> reads = new ArrayList<>();
-        Model model = spec.harry.quiescentChecker();
-        for (Long pd : new TreeSet<>(spec.harry.pds()))
+
+        for (Integer pkIdx : spec.pkGen.generated())
         {
-            reads.add(new HarryCommand(s -> "Harry Validate pd=" + pd  + state.commandNamePostfix(), s -> model.validate(Query.selectAllColumns(schema, pd, false))));
-            // as of this writing Accord does not support ORDER BY
-            if (!writeThroughAccord)
-                reads.add(new HarryCommand(s -> "Harry Reverse Validate pd=" + pd + state.commandNamePostfix(), s -> model.validate(Query.selectAllColumns(schema, pd, true))));
```

Comment:
what happened to reverse select?

Reply:
I have considered it to be an unnecessary aspect of the test in the given context. Main purpose of the test seems to be testing consistency, so it is important to verify the data is there after topology changes. Reading in reverse order should be tested separately. 

I have brought it back for now.

Reply:
> I have considered it to be an unnecessary aspect of the test in the given context

it breaks accord, so its useful =D

> Main purpose of the test seems to be testing consistency

Nope!  Main purpose is to put the cluster into *allowed* states and see what happens... throw random stuff at it and see what fires pop up...

> Reading in reverse order should be tested separately

In the context of this test it's its own `Command`!  Again, the focus on this test is mostly to to see happens when you do weird things... should reverse order break... no... can it? maybe.... so why not!